set.mm

```diff
@@ -752285,6 +752294,7 @@ with simplest antecedents (i.e., in the corresponding ordering of binary
 
 $( (End of Adhemar's mathbox.) $)
```

Comment:
I think  Adhemar's and  Igor Ieskov's mathboxes are misplaced: they should be moved up so that the mathboxes are alphabetically sorted.

Reply:
You're right. Can we do that in another PR ?

Reply:
> You're right. Can we do that in another PR ?

yes, of course.